contributing_to_docs/term_glossary.adoc

```diff
@@ -386,17 +386,17 @@ Pod are running at all times. Used by DeploymentConfigs.
 Do not use: replication controller(s)
 
 ''''
-=== Route
+=== Ingress
 
-Usage: Route(s)
+Usage: Ingress, Ingress Controller(s)
 
 OpenShift-specific API object that allows developers to expose Services through
 an HTTP(S) aware load balancing and proxy layer via a public DNS entry. The
-Route may further specify TLS options and a certificate, or specify a public
-CNAME that the OpenShift router should also accept for HTTP and HTTPS traffic.
-An administrator typically configures their router to be visible outside the
-cluster firewall, and may also add additional security, caching, or traffic
-controls on the Service content.
+Ingress Controller may further specify TLS options and a certificate, or specify
```

Comment:
```Ingress Controller```→``` `Ingress` resource ``` on this line.  (Technically, the `Ingress` resource may specify _the name of a secret that contains_ the certificate, whereas the `Route` resource specifies the certificate itself, but it is probably safe to gloss over that detail.)